tokio/src/sync/mpsc/chan.rs

```diff
@@ -288,8 +332,9 @@ impl<T, S> Chan<T, S> {
         // Push the value
         self.tx.push(value);
 
-        // Notify the rx task
+        // Notify the rx tasks
         self.rx_waker.wake();
+        self.not_ready_waker.wake();
```

Comment:
There are other places that wake `rx_waker`. Should the other places not be updated as well?